napari/layers/points/points.py

```diff
@@ -185,6 +185,8 @@ class Points(Layer):
 
     Extended Summary
     ----------
+    _property_choices : dict {str: array (N,)}
+        Possible values for the properties in Points.properties
```

Comment:
```suggestion
        Possible values for the properties in Points.properties.
        If ``properties`` is not provided at instantiation, will be undefined.
```

Reply:
Thank you for catching this, @tlambert03! I didn't intend to make it undefined. I've fixed it!